elegant/clean_timepoint_data.py

```diff
@@ -187,3 +187,26 @@ def remove_dead_timepoints(experiment_root, postmortem_timepoints, dry_run=False
             timepoints_after_death = timepoint_num - death_timepoint_index # positive values for timepoints after death
             if timepoints_after_death > postmortem_timepoints:
                 remove_timepoint_for_position(experiment_root, position, timepoint, dry_run=dry_run)
+
+def purge_images_from_experiment(experiment_root):
+    '''Removes all image files from experiment directory
+
+    This function is intended to be run after an experiment directory has been backed up
+    (or possibly after an issue with the experiment makes the data non-usable).
+    The function will remove all image files from position directories, while leaving
+    position_metadata files intact.
+
+    Parameters:
+        experiment_root: str/pathlib.Path to experiment root
+    '''
+    delete_confirmed = input(f'Removing all image files from {experiment_root}. Press y and enter to confirm; press enter to abort:')
+    if delete_confirmed != 'y':
+        return
+
+    experiment_root = pathlib.Path(experiment_root)
+    for subdir in experiment_root.iterdir():
+        if subdir.name.isnumeric():
```

Comment:
`isnumeric` is not a good way to identify the positions. There's no requirement that position names be numeric... indeed, for slides with multiple genotypes I would strongly recommend that they not be numeric. (Or you could do separate experiments, but I'd suggest one experiment with positions named `N2-nnn` and `spe-9-nnn` or something.)

Instead I'd suggest using `elegant.load_data.read_metadata()` to grab the positions from the metadata dictionary, or identifying position dirs as follows:
```python
for position_root in sorted(p.parent for p in experiment_root.glob('*/position_metadata.json')):
    # do something
```

Reply:
Thanks - I forgot about this. Identified by glob'ing.